### DISCUSSION TITLE
Plugin Conflicts

### DISCUSSION THREAD

**Discussion by goldfinder** [SkriptLang/Skript#6104] [Suggestions]
I have to ask, are you guys planning to fix Conflicts between plugins, like adding something to identify base skript against something like disky, or skript util?
IE:
`skGlow:make player glow red glow | VS |
make player glow red glow`
One is very strict, meaning if you don't have the plugin, it will not work, or spit out a error as such: "Addon not found: skGlow"

**Comment by moraedu** [comment]
Skript can't possibly know from where a syntax that haven't been registered comes from, at best something could be made with skUnity or Skripthub's API in order to recognize from where a syntax comes from, but that'd be a lot more work than it is worth and would have to be discussed among the owners of the respective services in order to not overwhelm their servers with requests.

So, in simpler terms, it is just not feasible.

**Comment by MinecraftMan1013number2** [comment]
If there is a conflict, you should reach out to the addon author. In this case, I am the author of SkGlow, so if there is an issue feel free to DM me. My discord is MinecraftMan1013
